docs/userGuide/tweakingThePageStructure.md

```diff
@@ -403,6 +403,65 @@ Alternatively, you can specify tags to render for a page in the front matter.
 ```
 </div>
 
-Tags in `site.json` will take precedence over the ones in the front matter.
+Tags in `site.json` will be merged with the ones in the front matter.
```

Comment:
Due to how they are merged, and with the additional wildcard feature, site.json will still "take precedence". This aspect needs to be better explained?